Product/Production/Adapters/General/CONNECTConsumerPreferencesProfileGUI/src/main/java/gov/hhs/fha/nhinc/adapter/cppgui/valueobject/PatientVO.java

```diff
@@ -26,12 +26,18 @@
  */
 package gov.hhs.fha.nhinc.adapter.cppgui.valueobject;
 
+
+import java.io.Serializable;
+
+import gov.hhs.fha.nhinc.adapter.cppgui.valueobject.PatientPreferencesVO;
```

Comment:
Again, check auto-formatting settings.  There should be a single line between packages, no lines between imports, and imports should be organized alphabetically.